### ISSUE TITLE
Configuration with Symfony 4?

### ISSUE THREAD

**Issue by karlbuckland** [getsentry/sentry-symfony#215]
I see that there is support for Symfony 4, but I the instructions are still for Symfony 3. I'm not sure how to configure for 4?

Obviously app/AppKernel.php no longer exists and app/config/config_prod.yml also no longer exists, so how do I enable the bundle?

**Comment by Jean85** [comment]
For Symfony 4, there's already a Symfony Flex recipe, so if you're using it you'll have the bundle automatically enabled, and the config file created for you.

If you're not using Flex, you'll have to enable the bundle in the `AppKernel` or `App\Kernel` class and create the configuration somewhere in your YAML files, depending on if you're using the older or newer file structure.

Happy to receive PRs to improve the readme in that regard!

**Comment by karlbuckland** [comment]
Aaah, ok. I didn't realise. I guess you put a section on the readme for Symfony 4 then and say it's using Flex? 

I can now see the sentry.yaml in /config/packages/. I've specified the DSN and run php bin/console sentry:test, which works, but normal exceptions/errors aren't getting in to Sentry?

**Comment by karlbuckland** [comment]
After digging around a great deal it appears to be because I had defined my own kernel event listener to handle exceptions so that I could intercept 404 requests. This probably means that the exceptions aren't bubbling up to sentry-symfony. When I started my project I thought this would be necessary but actually it no longer appears to be so I've restructured my code to remove the event listener and now everything is working as expected.

Thank you for your help!
